flow/connectors/postgres/cdc.go

```diff
@@ -501,7 +508,13 @@ func PullCdcRecords[Items model.Items](
 			if pkm.ServerWALEnd > clientXLogPos {
 				clientXLogPos = pkm.ServerWALEnd
 			}
-			pkmRequiresResponse = true
+			if pkmEmptyBatchThrottleEnabled {
+				if pkm.ReplyRequested || time.Since(lastEmptyBatchPkmSentTime) >= 1*time.Minute {
+					pkmRequiresResponse = true
+				}
+			} else {
+				pkmRequiresResponse = true
+			}
```

Comment:
```suggestion
			if pkmEmptyBatchThrottleEnabled || pkm.ReplyRequested || time.Since(lastEmptyBatchPkmSentTime) >= 1*time.Minute {
				pkmRequiresResponse = true
			}
```

Reply:
`pkmEmptyBatchThrottleEnabled` = false by default -> we want to not throttle

Reply:
right, I'm not thinking straight, long day